client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/PrescriptionLineEditTable.tsx

```diff
@@ -118,6 +118,7 @@ export const PrescriptionLineEditTable: React.FC<
       <Divider margin={10} />
       <Box
         style={{
+          maxHeight: 350,
```

Comment:
sorry @roxy-dao can you update this one too? please. I'm still getting the vertical scroll with 350

Reply:
I put it smaller to account for any messages too (if they pop up)